web/src/lib/actions/shortcut.ts

```diff
@@ -20,7 +20,7 @@ export const shouldIgnoreShortcut = (event: KeyboardEvent): boolean => {
     return false;
   }
   const type = (event.target as HTMLInputElement).type;
-  return ['textarea', 'text', 'date', 'datetime-local'].includes(type);
+  return ['textarea', 'text', 'date', 'datetime-local', 'email', 'password'].includes(type);
```

Comment:
I would be interested to know why we have this list at all. If it is an input or textarea and the shortcuts are not added directly to the element, I think we should just always ignore the event.

cc: @michelheusschen